### DISCUSSION TITLE
Broken workflow following commit-analyzer change

### DISCUSSION THREAD

**Discussion by kjporter** [semantic-release/semantic-release#3011] [Q&A] [answered]
Hello,
   Thanks in advance for your assistance. I'm having an issue with my workflow with a last successful run on 14 September. GitHub actions is throwing the following error:
```[5:42:35 PM] [semantic-release] › ℹ  Start step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[5:42:35 PM] [semantic-release] › ✘  Failed step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[5:42:35 PM] [semantic-release] › ✘  An error occurred while running semantic-release: TypeError: Cannot read properties of undefined (reading 'parserOpts')
    at default (file:///home/runner/work/z/z/node_modules/@semantic-release/commit-analyzer/lib/load-parser-config.js:32:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async analyzeCommits (file:///home/runner/work/z/z/node_modules/@semantic-release/commit-analyzer/index.js:31:18)
    at async validator (file:///home/runner/work/z/z/node_modules/semantic-release/lib/plugins/normalize.js:36:24)
    at async file:///home/runner/work/z/z/node_modules/semantic-release/lib/plugins/pipeline.js:38:36 {
  pluginName: '@semantic-release/commit-analyzer'
}
```
The only change that I've made to my workflow configuration was the node version (updated a recently unsupported v18.x to LTS).
It looks like the change that occurred in the commit-analyzer repo relative to this issue since my last successful run was 3ff9458 on 15 September "fix(config-loader): accounted for normalization of preset interfaces". https://github.com/semantic-release/commit-analyzer/commit/3ff94584f399738acb2075c6175df64e2ae181e5
Here is my plugin config for commit-analyzer from .releaserc.json:
```      
"@semantic-release/commit-analyzer",
      {
        "preset": "eslint",
        "releaseRules": [
          { "tag": "Breaking", "release": "major" },
          { "tag": "Build", "release": "minor" },
          { "tag": "Chore", "release": "minor" },
          { "tag": "Fix", "release": "patch" },
          { "tag": "New", "release": "minor" },
          { "tag": "Update", "release": "minor" },
          { "tag": "Upgrade", "release": "minor" }
        ]
      }
    ],
```
I've tried defining parserOpts in the plugin config, but it yields the same error. Any suggestions would be greatly appreciated.

**Comment by travi** [comment]
Have you updated to the latest version of the eslint preset?

**Reply by kjporter** [reply]
Hello @travi - thanks for your quick reply! I'm installing `conventional-changelog-eslint` from npm  as part of a GitHub workflow just prior to running `semantic-release`. I believe that it should be pulling the most recent version of `conventional-changelog-eslint`, but maybe I'm mistaken.

**Reply by travi** [reply, marked as answer]
Are you also installing the latest version of semantic-release?

**Reply by kjporter** [reply]
Thanks @travi - that was the issue. I must have been using a cached version of sematic-release. Really appreciate your help!
